drivers/i2s/Kconfig

```diff
@@ -7,7 +7,7 @@
 # I2S Drivers
 #
 menuconfig I2S
-	bool "I2S bus drivers"
+	bool "Inter-IC Sound (I2S) bus drivers"
```

Comment:
Late to the party, but shouldn't this be "Inter-Integrated Circuit Sound", not "Inter-IC Sound"?

Reply:
According to the standard it is "Inter-IC Sound (I2S) bus": https://www.nxp.com/docs/en/user-manual/UM11732.pdf